### PR TITLE
Import CGFloat APIs correctly

### PR DESCRIPTION
**Pull request by salinas-miguel**
Resolves rdar://88462147

### CODE REVIEW

**Comment by salinas-miguel** [comment]
@swift-ci please test

File: lib/PrintAsClang/DeclAndTypePrinter.cpp [hunk 1]
```diff
@@ -1478,6 +1478,10 @@ class DeclAndTypePrinter::Implementation
       specialNames[{ID_CoreGraphics, ctx.getIdentifier("CGFloat")}]
         = { "CGFloat", false };
 
+      Identifier ID_CoreFoundation = ctx.getIdentifier("CoreFoundation");
+      specialNames[{ID_CoreFoundation, ctx.getIdentifier("CGFloat")}]
```
Comment (beccadax) [inline lib/PrintAsClang/DeclAndTypePrinter.cpp:1482]: `ctx.Id_CoreFoundation` exists; you can use that instead.

(For that matter, so do `ctx.ID_CoreGraphics` and `ctx.Id_CGFloat`, so *all* of the `ctx.getIdentifier()` calls relating to CGFloat are unnecessary. I think there's been a little bit too much copy-and-paste programming...)

Comment (salinas-miguel) [reply lib/PrintAsClang/DeclAndTypePrinter.cpp:1482]: Thanks. I'll update those as well.

**Comment by salinas-miguel** [comment]
@swift-ci please test

**Comment by salinas-miguel** [comment]
@swift-ci please test

**Review by beccadax — APPROVED** [review]

**Pull request merged**